j/JCL.jcl

```diff
@@ -0,0 +1,16 @@
+//HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',CLASS=R,
+//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=S0JM
+//*
+//* PRINT "HELLO WORLD" ON JOB OUTPUT
+//*
+//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
+//*   AND WILL CAUSE A JCL ERROR
+//*
+//STEP0001 EXEC PGM=IEBGENER
+//SYSIN    DD DUMMY
+//SYSPRINT DD SYSOUT=*
+//SYSUT1   DD *
+HELLO, WORLD
```

Comment:
Wouldn't this [print `HELLO, WORLD` instead of `HELLO WORLD`](https://www.youtube.com/watch?v=XbuKDC_n3N4&t=649s)?